ct.yaml

```diff
@@ -12,7 +12,4 @@ chart-repos:
   - open-telemetery=https://open-telemetry.github.io/opentelemetry-helm-charts
 # Some of tests require longer than default timeout
 helm-extra-args: --timeout 800s --debug
-# We do not want to bump the Chart version for every single commit,
-# but only once we release
-check-version-increment: false
 validate-maintainers: false
```

Comment:
Since you added the `maintainers` field in Chart.yml maybe we can enable this as well (or just remove the line since it is `true` by default)?

Reply:
Initially I wanted to do that, but for some reason `ct` was complaining so I left this option here. It is definitely something we should do in the future.